src/main/java/net/atos/zac/signalering/SignaleringenJob.java

```diff
@@ -222,7 +222,7 @@ private ZoekParameters getOpenZaakMetBehandelaarZoekParameters(final Zaaktype za
         parameters.addFilter(FilterVeld.ZAAK_ZAAKTYPE_UUID, UriUtil.uuidFromURI(zaaktype.getUrl()).toString());
         parameters.addFilter(FilterVeld.ZAAK_BEHANDELAAR, NIET_LEEG.toString());
         parameters.addFilterQuery(ZAAK_AFGEHANDELD_QUERY, "false");
-        parameters.setRows(Integer.MAX_VALUE);
+        parameters.setRows(10000); //Should be enough, max is 100 on a page
```

Comment:
Same here. Better to make this into a class constant. Also, 10000 still seems very arbitrary. And 'should be enough' sounds like we don't really know how this works?